doc/news/changes/minor/20180809StefanKaessmairDanielArndt

```diff
@@ -0,0 +1,4 @@
+Improved: Triangulation::prepare_coarsening_and_refinement also takes care of
+limiting the level difference across periodic boundary faces.
+<br>
+(Stefan Käßmair, Daniel Arndt, 2018/08/09)
```

Comment:
Do we allow unicode in these changelogs? In any case, I think that Stefan prefers his name to be written in professional contexts as [`Kaessmair`](http://www.doi.org/10.1016/j.euromechsol.2014.03.007) (not that you were to know, of course).

Reply:
Yes, I think that we allow umlauts. At least previous, changelog entries have some.
Anyway, everything is easier without the potential struggle of dealing with them.